modules/archetypes/lib/policy_set_definitions/policy_set_definition_es_deploy_diag_loganalytics.tmpl.json

```diff
@@ -1045,7 +1045,7 @@
       },
       {
         "policyDefinitionReferenceId": "DataExplorerClusterDeployDiagnosticLogDeployLogAnalytics",
-        "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/Deploy-Diagnostics-DataExplorerCluster",
+        "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/Deploy-Diagnostics-DataExplorerCLuster",
```

Comment:
Need to fix name at source in [Azure/Enterprise-Scale](https://www.github.com/Azure/Enterprise-Scale)